assert.js

```diff
@@ -5,11 +5,39 @@
   * @param {string} key - The identifier for the parameter.
   *
   * @throws - If the value is not a string or is empty.
-  * @returns {undefined} - It just throws or you are ok.
+  * @returns {undefined} - No value is returned upon successful execution.
 */
 export function nonEmptyString(value, key) {
   if(!(value && typeof value === 'string')) {
     throw new TypeError(`"${key}" must be a non-empty string.`);
   }
 }
-export default {nonEmptyString};
+/**
+  * Ensures `parentCapabilities` has all required properties.
+  *
+  * @param {object} parentCapabilities - Contains the properties edvId and
+  *   edvRevocations.
+  * @param {string} edvId - The ID of the EDV that must be a URL
+  *   that refers to the EDV's root storage location.
+  * @param {object} hmac - A default HMAC API for blinding indexable
+  *   attributes.
+  * @param {object} keyAgreementKey - A default KeyAgreementKey API for
+  *   deriving shared KEKs for wrapping content encryption keys.
+  *
+  * @throws - If both of the required properties are undefined.
+  * @returns {undefined} - No value is returned upon successful execution.
+  */
+export function parentCapabilitiesValidator(
+  parentCapabilities, edvId, hmac, keyAgreementKey
```

Comment:
Please use named parameters whenever possible.  I see that the only other existing assert method is patterned after assert-plus with ordered parameters.  I think named parameters is more appropriate for this validator.